Extension/src/LanguageServer/client.ts

```diff
@@ -464,7 +480,7 @@ class DefaultClient implements Client {
                 errorSquiggles: settings.errorSquiggles,
                 dimInactiveRegions: settings.dimInactiveRegions,
                 textMateColorization: settings.textMateColorization,
-                enhancedColorization: settings.enhancedColorization,
+                enhancedColorization: abTestSettings.UseEnhancedColorization,
```

Comment:
use settings.enhancedColorization here. We change the default, but should still use whatever VS Code reports to us here.